src/main/java/io/r2dbc/postgresql/PostgresqlConnectionFactory.java

```diff
@@ -73,6 +76,10 @@ public PostgresqlConnectionFactoryMetadata getMetadata() {
         return PostgresqlConnectionFactoryMetadata.INSTANCE;
     }
 
+    public PostgresqlConnectionConfiguration getConfiguration() {
```

Comment:
Let's keep the configuration internal and not expose it.

Reply:
True. It was added for the tests to check the configuration, but I think making it package-private will be fine. Right?